marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/entity-driven/test-data/entities/Customer.entity.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
I highly recommend slimming down the test data to the bare minimum needed. For example, only include properties used for matching - anything else is just noise that makes the test more difficult to understand. Same thing goes for test data - only insert the documents you need to verify that e.g. 2 docs match, but a 3rd doc does not match.